classes/api/crud.php

```diff
@@ -253,7 +253,9 @@ public function create_item(WP_REST_Request $request)
      */
     public function update_item(WP_REST_Request $request)
     {
-        return $this->not_yet_response();
+
+        \Caldera_Forms_API_Forms::save_form( $request );
```

Comment:
@New0 This is a base class extended by entries and forms, so it can't just save a form.

The reason I didn't extend this method in the forms API is that there was no way to change the request arguments.


- https://github.com/CalderaWP/Caldera-Forms/pull/3576/files#diff-1e2e9b0f9918e8d326a8e6306d158274R397
- https://github.com/CalderaWP/Caldera-Forms/pull/3576/files#diff-50f33cdd5ab83a6c8d8a844c07015635R32

Reply:
@New0 you wrote 

> The Save Form process still calls update_item() in api/crud.php instead of save_form() in api/forms.php." 

This statement is correct, and is what I intened. It smells bad, but it's a work around for a decision I made years ago that in hindsight wasn't great, and it works.